scipy/io/_mmio.py

```diff
@@ -20,6 +20,18 @@
 
 __all__ = ['mminfo', 'mmread', 'mmwrite', 'MMFile']
 
+"""
+Maximum number of digits for `mmwrite` function.
+This can be calculated by below python codes:
+eps = np.finfo(np.float64).eps
+max_precision = np.floor(-np.log10(eps)).astype(int)
+"""
+MAX_PRECISION = 15
```

Comment:
Should this parameters be the same across all environments? Or would it be better to calculate them dynamically?

Reply:
I would make this 16. The standard eps for 64bit floats is around 2e-16, which is closer to 16 digits than 15.